### PR TITLE
Should fix all the new mutations being quirked up and/or goated with the sauce

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

formatting was wrong bc genetics code is wacky
also removed all the conditionals bc we can expect them to exist
also made the marrow one not stack your hunger mod bc that would be spicy instead it just burns nutriment

# Changelog

:cl:  
bugfix: new genetics mutations should all work properly now
/:cl:

### CODE REVIEW

File: code/datums/mutations/body.dm [hunk 1]
```diff
@@ -472,16 +472,18 @@
...
 		ADD_TRAIT(owner, TRAIT_HARDLY_WOUNDED, "genetics")
 
 /datum/mutation/human/densebones/on_losing(mob/living/carbon/human/owner)
-	. = ..()
-	owner?.physiology?.armor?.melee += 5
-	owner?.physiology?.armor?.wound += 10
+	if(..())
+		return
+	owner.physiology.armor.melee += 5
+	owner.physiology.armor.wound += 10
```
Comment (tattax) [inline code/datums/mutations/body.dm]:
```suggestion
	owner.physiology.armor.melee -= 5
	owner.physiology.armor.wound -= 10
```

**Review by ToasterBiome — APPROVED** [review]

File: code/datums/mutations/body.dm [hunk 2]
```diff
@@ -472,16 +472,18 @@
...
 	if(GET_MUTATION_POWER(src) > 1)
 		ADD_TRAIT(owner, TRAIT_HARDLY_WOUNDED, "genetics")
 
 /datum/mutation/human/densebones/on_losing(mob/living/carbon/human/owner)
-	. = ..()
-	owner?.physiology?.armor?.melee += 5
-	owner?.physiology?.armor?.wound += 10
+	if(..())
+		return
+	owner.physiology.armor.melee += 5
```
Comment (adamsong) [inline code/datums/mutations/body.dm]: Shouldn't this and the next one be -=

**Review by adamsong — CHANGES_REQUESTED** [review]

**Review by adamsong — APPROVED** [review]
Looks good

**Comment by GraveHat** [comment]
Thank you I am in deep pain.

File: code/datums/mutations/body.dm (hunk 2 above)
Comment (GraveHat) [reply code/datums/mutations/body.dm]: spotted this issue while I was working on a fix myself.

**Pull request merged**